<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QGroupBox" name="loginGroupBox">
      <property name="title">
       <string>登录区:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="userLabel">
                 <property name="text">
                  <string>账号:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="userLineEdit">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="loginButton">
                 <property name="text">
                  <string>登录</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="passwdLabel">
                 <property name="text">
                  <string>密码:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="passwdLineEdit">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="inputMask">
                  <string/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="logoutButton">
                 <property name="text">
                  <string>登出</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="nowUserLabel">
               <property name="text">
                <string>当前账号:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="usedLabel">
               <property name="text">
                <string>已用流量:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="totalLabel">
               <property name="text">
                <string>全部流量:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="leftLabel">
               <property name="text">
                <string>剩余流量:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="noticeLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="defaultButton">
           <property name="text">
            <string>设为默认账号</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="infoGroupBox">
      <property name="title">
       <string>设置区:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="usersHelp">
         <property name="text">
          <string>双击左边的行号可以切换账号:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QTableWidget" name="userTableWidget">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="columnCount">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="adduserButton">
             <property name="text">
              <string>添加</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="delUserButton">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>当流量低于:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="switchSpinBox"/>
           </item>
           <item>
            <widget class="QCheckBox" name="autoChange">
             <property name="text">
              <string>自动切换</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sharedButton">
      <property name="text">
       <string>分享</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>userLineEdit</tabstop>
  <tabstop>passwdLineEdit</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>logoutButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
